### PR TITLE
Pilot packages - cold start

### PR DESCRIPTION
**Pull request by Eomm**
Hi,
based on the meeting of yesterday I send this PR for the issue #142 

The list is not completed, we are waiting for others feedback/comment/approvals, meanwhile, we could start with confirmed ones.

I was thinking if is it necessary to find out a "WG package-maintenance reference" per pilot package, not only for the module?

### CODE REVIEW

File: pilots/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+[MQTT]: https://github.com/mqttjs/MQTT.js
+
+*Ordered per joining time.
+
+
+## Approach
+
+The baseline to start with this work is:
+
++ Prepare a [survey](./SURVEY.md) to collect more informations as possible
```
Comment (mcollina) [inline pilots/README.md]: How will the survey delivered? Should we send a PR?

Comment (Eomm) [reply pilots/README.md]: Yes, the maintainer could send the reply with a PR (maybe there will be the necessity of little clarifications). But of course, the approval will be more like a Q&A.

Comment (mcollina) [reply pilots/README.md]: This should be clarified here.

File: pilots/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
++ give to all the maintainers the possibility to use them
++ provide the resources to choose wisely the dependencies for business
++ improve the quality and stability of the packages in the Node.js ecosystem
+
+
+## List
+
+| Package* | Maintainer Reference |
+|----------|----------------------|
+| [Express]     | [@wesleytodd](https://github.com/wesleytodd)
```
Comment (wesleytodd) [inline pilots/README.md]: To be clear here, I think we should change this to Organization, or Ecosystem.  It is not the `express` package specifically that needs attention.  It is the entire ecosystem of modules that make up what people think of when they say "express".  This includes all the modules in [jshttp](https://github.com/jshttp) and [pillarjs](https://github.com/pillarjs) as well as the ones directly in the [express org](https://github.com/expressjs).

Comment (Eomm) [reply pilots/README.md]: Sorry if it is a silly question but, in this case, our reference will be anyway Express right?

File: pilots/README.md (hunk 1 above)
Comment (Eomm) [reply pilots/README.md]: Done

File: pilots/README.md (hunk 2 above)
Comment (wesleytodd) [reply pilots/README.md]: Well I am not sure there will be much to apply in the express repo proper.  My guess is most of the recommendations and changes will be in the other repos, which is why I mentioned it.

Comment (Eomm) [reply pilots/README.md]: Clear, I'm thinking only how to match this use-case with the "SURVEY.md", I think I should add some questions like:
"which problems are due your dependencies?"

Comment (wesleytodd) [reply pilots/README.md]: FWIW, this is the kind of thing I am referring to when I say "not express, but express repos".

https://github.com/pillarjs/hbs/issues/162

There are a bunch of modules which Doug is maintaining or in some way managing.  See all the repos in `pillarjs` and `jshttp` as mentioned above.  These are the ones the Express TC is responsible for, and are short staffed for.

**Comment by mhdawson** [comment]
@wesleytodd, @mcollina look good to you, looks like your comments were addressed.

File: pilots/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
++ provide the resources to choose wisely the dependencies for business
++ improve the quality and stability of the packages in the Node.js ecosystem
+
+
+## List
+
+| Organization* | Maintainer Reference |
+|---------------|----------------------|
+| [Express]     | [@wesleytodd](https://github.com/wesleytodd)
+| [MQTT]        | [@mcollina](https://github.com/mcollina)
```
Comment (mcollina) [inline pilots/README.md]:
```suggestion
| [MQTT.js]        | [@mcollina](https://github.com/mcollina)
```

File: pilots/README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+[Express]: https://github.com/expressjs/express
+[MQTT]: https://github.com/mqttjs/MQTT.js
```
Comment (mcollina) [inline pilots/README.md]:
```suggestion
[MQTT.js]: https://github.com/mqttjs/MQTT.js
```

**Comment by mhdawson** [comment]
@Eomm if you can accept the suggested  changes we can get this landed.

**Comment by Eomm** [comment]
@mhdawson done 👍

**Pull request merged**